frontend/src/citizen-frontend/App.tsx

```diff
@@ -94,33 +94,17 @@ const Content = React.memo(function Content() {
 
   const { modalOpen } = useContext(OverlayContext)
 
-  const mainRef = useRef<HTMLDivElement>(null)
-  const scrollMainToTop = useCallback(
-    () =>
-      scrollElementToPos(mainRef.current, {
-        top: 0,
-        left: 0,
-        behavior: 'auto'
-      }),
-    []
-  )
-
   return (
     <FullPageContainer>
+      <ScrollToTop />
```

Comment:
This might require the fewest keystrokes, but the condition whether to scroll or not is hidden inside the component.

Maybe `<Route path="..." element={<ScrollToTop><MapPage /><ScrollToTop>} />` would be a good middle ground?

Reply:
I agree that it's ugly, but in my opinion it's better that it's enabled everywhere by default. But I guess it's very likely that if a new route is added it's wrapped in `<ScrollToTop>` like other routes.